internal/debugger/handler_debug.go

```diff
@@ -104,6 +105,17 @@ func (s *Server) handleDebug(ctx context.Context) http.HandlerFunc {
 				return fmt.Errorf("failed to get all export configs: %w", err)
 			}
 			resp.ExportConfigs = exportConfigs
+
+			queue(&wg, errCh, func() error {
+				for _, ec := range exportConfigs {
+					exportBatchFiles, err := exportDB.LookupExportFiles(ctx, ec.ConfigID, 4*time.Hour)
```

Comment:
Not technically part of this PR but something I just noticed: does the "4" need to be configurable or based on some existing configuration?